kmk/kmk_keyboard.py

```diff
@@ -87,7 +87,7 @@ def _print_debug_cycle(self, init=False):
             debug(f'keys_pressed={self.keys_pressed}')
 
     def _send_hid(self):
-        if self._hid_send_enabled:
+        if self._hid_send_enabled and runtime.usb_connected:
```

Comment:
Wouldn't that only address USB connections and probably break bluetooth?
We may even want to catch connection errors in the HID classes, not `KMKKeyboard`.
Definitely needs hardware testing.

Reply:
@xs5871 Ho good point! Haven't thought of bluetooth... Fixed